test/subroutine/base_test.rb

```diff
@@ -318,5 +318,11 @@ def test_actioncontroller_parameters_can_be_provided
       assert_equal raw_params, op.params
     end
 
+    def test_inspect_is_pretty
+      op = SignupOp.new({ email: "foo@bar.com", password: "password123!" })
+      oid = format('%x', (op.object_id << 1))
+      assert_equal "#<SignupOp:0x#{oid} email: \"foo@bar.com\", password: \"password123!\">", op.inspect
```

Comment:
any concern of this leaking param values like this to logs and such? should #inspect be param filter aware?

Reply:
I could definitely see this being enhanced by providing a way to generically add a filter class to your config as long as it quacks in a certain way.